buildbot_effects/__init__.py

```diff
@@ -203,6 +207,8 @@ def run_effects(
         "--tmpfs",
         "/tmp",  # noqa: S108
         "--tmpfs",
+        "/var/tmp",  # noqa: S108
+        "--tmpfs",
```

Comment:
What is /var/tmp used for here?

Reply:
Skopeo uses that and doesn't appear to use `$TMPDIR` for that location; I see now that it's possible to override with the `--tmpdir` option, but if one tool gets this wrong, chances are many other tools will.

Reply:
Oh, I see:

```
17-// DO NOT change this, instead see unixTempDirForBigFiles above.
18:const builtinUnixTempDirForBigFiles = "/var/tmp"
```

This is very odd behavior. Do they expect people to run this as root?

Reply:
On the other hand, you are not really solving their concern, because you make the directory a tmpfs again. Should we not rather make `/var` be backed by some actual filesystem?

Reply:
I'm not sure there's a problem to be solved besides making it available - skopeo requires some directory to stage its "big" files, and /var/tmp works just as well as /tmp for those purposes (they're both mode `1777` on non-nixos systems).

If a tool needs a persisted cache, it's probably better to use that "state file" mechanism exposed by hci effects...

Reply:
From what I can read, it actually unpacks docker images, there. So those can be bigger than your RAM. I have the feeling if applications reach out to write to /var/tmp specifically they must have a good reason to not use /tmp.

Reply:
I also want to keep the possibility of using impure derivations at some point instead of bubblewrap because that's more cross-platform. We don't have /var/tmp there as well.

Reply:
Hm, yeah. I mean, I'm happy to back this out (it's not _necessary_ for my use case), but it feels like `/var/tmp` is a thing that tools might expect of a working system. There should be a way to get one (one way or another) but that can remain a follow-up task after effects land.

Reply:
Backed out.